src/sentry/models/userreport.py

```diff
@@ -32,3 +32,18 @@ class Meta:
         unique_together = (('project', 'event_id'), )
 
     __repr__ = sane_repr('event_id', 'name', 'email')
+
+    def notify(self):
+        from django.contrib.auth.models import AnonymousUser
+        from sentry.api.serializers import (
+            serialize, ProjectUserReportSerializer
+        )
+        from sentry.tasks.signals import signal
+
+        signal.delay(
```

Comment:
secondary benefit here -- this will fit nicely into the pubsub model in the future